src/web/multi-target.md

```diff
@@ -0,0 +1,35 @@
+# Multi-Target (targeting both native and WASM)
```

Comment:
The delta between this multi-target and the below single target isn't very big and users will likely often want the multi-target so it might be valuable to just merge them into one set of directions. Then users can just remove the native targets if they don't want them.

Reply:
I agree, but this is made a bit more difficult by the `bevy_webgl2_app_template` template for `cargo-make` we use in the instructions, which currently still relies on platform-specific features. There's already an [issue](https://github.com/mrk-its/bevy_webgl2_app_template/issues/5) open upstream to rectify this, once Rust 1.51 releases.

Alternatively we split off the `cargo-make` sections entirely first. More on that below.